backtracking/fibonacci.py

```diff
@@ -0,0 +1,14 @@
+"""the problem is write a function who represent the sum of 0,1,2,3,n elements ,where n is a argument from the function.Using backtracking when save the value of manys fibonacci call.
+"""
+def backtracking_fibonacci(n) :
```

Comment:
It is a python convention to use 4 spaces for indentation. The variable name `n` is not descriptive. Also, please consider adding type hints to the function:

```python
def backtracking_fibonacci(num_of_elements: int) -> int:
    ...
```